### DISCUSSION TITLE
Unable to add Open AI API Key

### DISCUSSION THREAD

**Discussion by multipletwigs** [zed-industries/zed#6752] [Feature Requests]
Hey there! Loving the editor so far, however I can't seem to add in my own OpenAPI Key through the assistant panel. Any help is appreciated!

![image](https://github.com/zed-industries/zed/assets/63769232/f3cc3880-966f-48c6-a390-bbe7ee0d94bf)
![image](https://github.com/zed-industries/zed/assets/63769232/034cc1ef-12ba-4a1a-9a62-97d80b1d34e2)

**Comment by mrnugget** [comment]
Hey! When you have focused the assistant panel and hit `cmd-p`, do you see the `assistant: reset key` action in there?

**Comment by mikaelsouza** [comment]
Hey, I think what @mrnugget said is correct, but you can also check out [this issue](https://github.com/zed-industries/zed/issues/6685). There a few explanations and possible solutions there for another problem, but maybe it can be helpful.

**Comment by multipletwigs** [comment]
@mrnugget @mikaelsouza Ah! Thanks for the help, I think I'll be bringing the discussion there!
